benchmarks/bench_methods_plot.py

```diff
@@ -278,11 +288,13 @@ def plot_barplot(all_results_df, threshold, normalize_method, save_name):
             all_results_df = data_frame
 
         # plot the results of each setting
-        plot_convergence(data_frame, threshold=1e-2, normalize_method='last',
+        plot_convergence(data_frame, threshold=threshold,
+                         normalize_method=normalize_method,
                          save_name=load_name[:-4])
         plt.close('all')
 
     # plot the aggregation of all results
-    plot_barplot(all_results_df, threshold=1e-2, normalize_method='last',
+    plot_barplot(all_results_df, threshold=threshold,
+                 normalize_method='last',
```

Comment:
I am not a big fan of `normalize_method='last'`. Could you use `best` or `none`?

Reply:
I've changed it, however I realised that for `None`, barplot returns None. 

And for `best`, it raises exception:
```
Traceback (most recent call last):
  File "bench_methods_plot.py", line 299, in <module>
    save_name=str(figures_dir / 'all'))
  File "bench_methods_plot.py", line 189, in plot_barplot
    pobj = normalize_pobj(pobj, None, normalize_method)
  File "bench_methods_plot.py", line 31, in normalize_pobj
    assert best_pobj is not None
AssertionError

```